docs/backup-restore.md

```diff
@@ -3,30 +3,63 @@
 Since Scalar DB provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
 This document sets out some guidelines for backing up and restoring the databases that Scalar DB supports.
 
-## Cassandra
+## Scalar DB server
+
+Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
+You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
+* Take backups of your databases during the paused duration.
+* Restore to a point in the paused duration with point-in-time-restore (PITR) feature.
+
+## How to create transactionally-consistent backup
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on a non-transactional database is to use the [Scalar DB server](https://github.com/scalar-labs/scalardb/tree/master/server) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
```

Comment:
Mostly saying the same thing as above?

The above two ways should come here?

Reply:
Sorry for the inconvenience.
Document updated with a new structure.